foyer/atomtyper.py

```diff
@@ -66,7 +68,7 @@ def find_atomtypes(structure, forcefield, max_iter=10):
             except IndexError:
                 try:
                     atomic_num = next(atom.find_data('atomic_num')).children[0]
-                    element = Element[atomic_num]
+                    element = pt.Element[atomic_num]
```

Comment:
A question/comment.  I'm not saying this needs to be addressed in this PR, just something that came to mind while looking at this section of code.  

In the future, are we going to make a gmso function to get the elements so the gmso topology is stand-alone from the parmed structure?

Reply:
yes, that's our intention. This issue is addressed in the other PR https://github.com/mosdef-hub/foyer/pull/395